airflow/models/connection.py

```diff
@@ -208,8 +208,8 @@ def get_uri(self) -> str:
         """Return connection in URI format"""
         if '_' in self.conn_type:
             self.log.warning(
-                f"Connection schemes (type: {str(self.conn_type)}) "
-                f"shall not contain '_' according to RFC3986."
+                "Connection schemes (type: %s) shall not contain '_' according to RFC3986.",
+                str(self.conn_type),
```

Comment:
```suggestion
                self.conn_type,
```

The logging framework stringify things automatically.

Reply:
🤦 Yep, thanks.